lib/migration.js

```diff
@@ -517,29 +378,9 @@ function mixinMigration(PostgreSQL) {
     return colDefault ? (' DEFAULT ' + colDefault) : '';
   };
 
-  /*!
-   * Find the column type for a given model property
-   *
-   * @param {String} model The model name
-   * @param {String} property The property name
-   * @returns {String} The column type
-   */
-  PostgreSQL.prototype.columnDataType = function(model, property) {
```

Comment:
Hmm, why changing this function from `columnDataType` to `buildColumnType`?
we need both of them.